src/draw/nema_gfx/lv_draw_nema_gfx_label.c

```diff
@@ -267,14 +446,16 @@ static void _draw_label_iterate_characters(lv_draw_unit_t * draw_unit, const lv_
         pos.y += dsc->hint->y;
     }
 
-    uint32_t line_end = line_start + lv_text_get_next_line(&dsc->text[line_start], font, dsc->letter_space, w, NULL,
-                                                           dsc->flag);
+    uint32_t remaining_len = dsc->text_length;
```

Comment:
Why do we need to implement `_draw_label_iterate_characters` here? Why not use the general `lv_draw_label_iterate_characters` function?

Reply:
@FASTSHIFT we had to do some changes inside those functions mainly for performance improvement reasons.

Reply:
What exactly has been modified? Can the optimization be put into the public part? 
Because the modification of the public part may not be synchronized to the private modification in time, it will take a lot of time to deal with code consistency issues.

Reply:
Yes you are right, we are aware of that. We are looking for other ways to achieve those optimizations, avoiding the modification of the public parts.